### PR TITLE
Week5 Assignments

### PR DESCRIPTION
**Pull request by pradeepmalvi**

### CODE REVIEW

File: Week-5/Exercise-5.1/script.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const doTheTask = (taskNumber) => {
```
Comment (sharmajitesh07) [inline Week-5/Exercise-5.1/script.js:1]: I think this is not what assignment is asking. We need to create a custom function which uses both async/await and generators. The mixed function should attempt to call three task. Go over below link to understand better

https://www.pluralsight.com/guides/using-asyncawait-with-generator-functions

Comment (sharmajitesh07) [inline Week-5/Exercise-5.1/script.js:1]: I think this is not what assignment is asking. We need to create a custom function which uses both async/await and generators. The mixed function should attempt to call three task. Go over below link to understand better

https://www.pluralsight.com/guides/using-asyncawait-with-generator-functions

File: Week-5/Exercise-5.4/calculator.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const mathOperations = {
+  sum: function (a, b) {
```
Comment (sharmajitesh07) [inline Week-5/Exercise-5.4/calculator.js]: why do we still not use arrow functions?

Comment (sharmajitesh07) [inline Week-5/Exercise-5.4/calculator.js]: why do we still not use arrow functions?

File: Week-5/Exercise-5.1/script.js (hunk 1 above)
Comment (pradeepmalvi) [reply Week-5/Exercise-5.1/script.js:1]: Done, Please check

File: Week-5/Exercise-5.4/calculator.js (hunk 2 above)
Comment (pradeepmalvi) [reply Week-5/Exercise-5.4/calculator.js]: Updated, Please check

**Pull request merged**